spec/inputs/http_spec.rb

```diff
@@ -145,6 +146,20 @@ def do_post
       end
     end
 
+    context "with json_lines codec without final delimiter" do
+      subject { LogStash::Inputs::Http.new("port" => port, "codec" => "json_lines") }
```

Comment:
can't we test this with `line` codec so we don't have to rely on a new feature of json_lines?

Reply:
yeah I guess so.